Chapter_2/2_10_DisturbedClosedOrbits.tex

```diff
@@ -130,47 +130,24 @@ \section{Disturbed closed orbits}\label{sec:2.10}
  from the design orbit.
 
  \subsection{Closed orbit correction}
-The design orbit is modified to a new closed orbit $x_c$ due to field errors, as discussed previously. The expression of this closed orbit is given in terms of different field errors $\delta G(s_j)$ at different positions $s_j$
-by the following:
 
- \begin{align*}
- x_c(s) = \dfrac{\sqrt{\beta(s)}}{2\sin\pi\nu} \sum_j \delta G({s}_j) \sqrt{\beta({s}_j)}\cos\{ |\varphi(s) - \varphi({s}_j)| - \pi\nu \}.
- \end{align*}
-
- Since orbit distortions can change beam parameters such as aperture for betatron oscillations it is commom to include a set of magnets for the correction of disturbed orbits. First of all, we need to know the beam position and only after
- that apply some correction on the orbit. The beam position is measured by devices called beam position monitors (BPM, for short). Suppose there are a number $m$ of BPMs over the storage ring, thus there will be $2m$ position measurements of displaced close orbits (horizontal and vertical), which we can organize as a column-vector with $2m$ components.
-
- \begin{align*}
- \mathbf{x} = \left(x_1,x_2,\ldots,x_{2m}\right).
- \end{align*}
-
- Suppose the strength of these small fields due to the $j$-th corrector is $\Delta \theta_j$ and over the storage rings are $n_x$ horizontal corrector magnets and $n_y$ vertical ones.
- Since we consider the field kick given by the corrector as a small perturbation, we already know the expression for the closed orbit displacement at position $s_i$ produced by all correctors:
-
- \begin{align*}
- \Delta x_i = \dfrac{\sqrt{\beta_i}}{2\sin\pi\nu} \sum_{j=1}^{n_x + n_y} \Delta \theta_j \sqrt{\beta_j}\cos\{ |\varphi_i - \varphi_j| - \pi\nu \}.
- \end{align*}
-
- Notice that the previous equation can be rewritten in form $\Delta x_i = \sum_{j=1}^{n_x + n_y}M_{ij}\Delta\theta_j$, which is nothing but a vectorial equation
+ The orbit distortion in the presence of field errors $\delta G(s_j)$ at different positions $s_j$ along the ring is given by:
+	\begin{align*}
+		u_c(s_i) = \dfrac{\sqrt{\beta_u(s_i)}}{2\sin\pi\nu} \sum_j \delta G({s}_j) \sqrt{\beta_u({s}_j)}\cos\{ |\varphi(s_i) - \varphi({s}_j)| - \pi\nu \},
+	\end{align*}
+	where $u$ may represent the $x$ or $y$ coordinate.  The orbit distortions $\Delta u(s_i)$ can be measured in localized positions $s_i$, where the Beam Position Monitors (BPMs) are installed. The equation above can be cast in the form $\Delta u_i = \sum_{j} M_{ij}^{uu} \Delta \theta_{j}^{u}$. Therefore, one can define the orbit response matrix, whose elements can be recognized as:
```

Comment:
It is not clear at this points that the field errors are created by the correctors, so it may confuse the reader to talk about response matrix now.